### DISCUSSION TITLE
What does git clone do?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#241] [Q&A] [answered]
Short answer.

**Comment by montasim** [comment, marked as answer]
Git clone allows you to create a local copy of the remote GitHub repository. Once you clone a repo, you can make edits locally in your system rather than directly in the source files of the remote repo

**Comment by ankit-akash** [comment]
It makes a copy file and saves in local system which later you can change it and push to github.

**Reply by montasim** [reply]
@ankit-akash try answering the questions that are not answered yet. That way you will get a badge.
You can filter the unanswered questions from here: https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions?discussions_q=is%3Aunanswered
